schedule/forms.py

```diff
@@ -1,13 +1,21 @@
 from django.forms import ModelForm
 from django.contrib.auth.forms import UserCreationForm
-from .models import Selection, MyUser
+from django.db.models import Count
+
+from .models import Selection, MyUser, Course
 
 
 class SelectionForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(SelectionForm, self).__init__(*args, **kwargs)
+        qs = Course.objects.annotate(scheds=Count('schedule')).exclude(scheds=0)
```

Comment:
I'm against that, such that you can already prepare your course list even if they are not scheduled yet

Reply:
Hum, good point :)
But then in a course disappears from a year to another it still stays in the list.
Could we then wipe the course list at the beginning of the year ?

Reply:
Commit reverted :chicken: